docs/source/gateway.mdx

```diff
@@ -63,6 +63,20 @@ As a recommended alternative, you can use the [Rover CLI](https://www.apollograp
 
 For an example of this, see the [Federation quickstart](./quickstart/).
 
+## Updating
+
+To update the `@apollo/gateway` package to the latest version:
+
+```
+npm update @apollo/gateway --latest
+```
+
+Similarly, to update to a specific version:
+
+```
+npm update @apollo/gateway@0.34.0
```

Comment:
I'm also not sure how providing a `@` range to the `npm update` command behaves in practice (I usually do this with `npm install pkg@version`).

Worth noting that such an update may be constrained by the range rules (e.g., the `^`, `~`, or lacking specifier) in the `package.json`.  See the [example section](https://docs.npmjs.com/cli/v7/commands/npm-update#example) of the `npm update` command.